README.md

```diff
@@ -14,7 +14,7 @@
 
 # Live Demo
 
-[Demo](Link)
+[Demo](https://jellywiz.github.io/Leaderboard/)
 
 ## Getting Started
```

Comment:
- **Please make your README file look professional:** It should contain all that's needed for running the app with the API(s) it consumes. 
**Hint**: How to make the project run on your local machine(You will need to set up or install webpack in your machine and also talk about the installation of Package.Json)
-
-
![Capture](https://user-images.githubusercontent.com/63972670/202224943-1de9c795-60cd-43f3-9840-889238dbd299.JPG)

Reply:
so i should delete this installation part? @OLIPLICHE